src/DotNetBoilerplate.Infrastructure/DAL/Handlers/Events/GetEventByIdHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DotNetBoilerplate.Application.Events;
+using DotNetBoilerplate.Application.Events.Responses;
+using DotNetBoilerplate.Core.Events;
+using DotNetBoilerplate.Infrastructure.DAL.Contexts;
+using DotNetBoilerplate.Shared.Abstractions.Queries;
+using Microsoft.EntityFrameworkCore;
+
+namespace DotNetBoilerplate.Infrastructure.DAL.Handlers.Events;
+
+internal sealed class GetEventByIdHandler(
+    DotNetBoilerplateReadDbContext dbContext
+    )
+: IQueryHandler<GetEventByIdQuery, EventInfoResponse>
+{
+    public async Task<EventInfoResponse> HandleAsync(GetEventByIdQuery query)
+    {
+        var @event = await dbContext.Events
+            .AsNoTracking()
```

Comment:
Też możemy od razu tu select wewalić ale jak tego nie lubisz to nie zmuszam, działanie końcowe takie samo